idl_parser/PRESUBMIT.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2013 The Chromium Authors. All rights reserved.
```

Comment:
This also doesn't do anything on GitHub, perhaps `git cl presubmit` can run this, otherwise I would just remove it.

Reply:
Done.